build.gradle

```diff
@@ -31,3 +31,21 @@ else {
 	throw new GradleException("GHIDRA_INSTALL_DIR is not defined!")
 }
 //----------------------END "DO NOT MODIFY" SECTION-------------------------------
+
+import org.gradle.internal.os.OperatingSystem
+
+def sleigh_path
+if (OperatingSystem.current() == OperatingSystem.WINDOWS)
+	sleigh_path = "/support/sleigh.bat"
+else
+	sleigh_path = "/support/sleigh"
+
+def sleigh = new File(ghidraInstallDir).getCanonicalPath() + sleigh_path
+task runSleigh(type:Exec) {
+    commandLine sleigh, "-a", "data/languages"
```

Comment:
Is compiling the `sla` files required? Doesn't Ghidra do it when you load the extension?

Reply:
It does now. I don't think ghidra did initially. Until any java code is added build.gradle shouldn't be necessary.

Reply:
Ah, perhaps. I will check on an older version. The gradle file is used to package the extension in to a zip too.